source/MRMesh/MRMeshDecimateCallbacks.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "MRMesh/MRMesh.h"
+
+namespace MR
+{
+// there is a vertex parameter that needs to be recalculated when collapsing the edge
```

Comment:
callback that is called when before edge is collapsed (i.e. in decimateMesh) and changes moved vertex attribute to correct value. Useful to update vertices based attributes like uv coordinates or verts colormaps